src/Sfa.Tl.Matching.Functions/EmployerReferralEmail.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Sfa.Tl.Matching.Application.Interfaces;
+using Sfa.Tl.Matching.Data.Interfaces;
+using Sfa.Tl.Matching.Domain.Models;
+using Sfa.Tl.Matching.Functions.Extensions;
+using Sfa.Tl.Matching.Models.Dto;
+using Sfa.Tl.Matching.Models.Enums;
+
+namespace Sfa.Tl.Matching.Functions
+{
+    public class EmployerReferralEmail
+    {
+        [FunctionName("SendEmployerReferralEmails")]
```

Comment:
Merge this with the ProviderReferralEmail, everywhere else they are together so might as well keep them together here

Reply:
Merged into a single class file "ReferralEmails.cs"